docs/maintain/databases.rst

```diff
@@ -339,6 +339,50 @@ Show running queries:
 
 See the `pg_stat_activity <https://www.postgresql.org/docs/11/monitoring-stats.html#PG-STAT-ACTIVITY-VIEW>`__ table's documentation.
 
+.. _pg-recover-backup:
+
+Restore from backup
+-------------------
+
+PostgreSQL databases are backed up offsite. Backup and restoration are managed by `pgBackRest <https://pgbackrest.org/>`__.
+These are the main commands for working with pgbackrest.
+
+.. note::
+
+   For more information on setting up backups, see :ref:`pg-setup-backups`.
+
+The stanza name is defined in pillar ``postgres:backup:identifier``.
+You can also find it in the pgbackrest config ``/etc/pgbackrest/pgbackrest.conf``.
+
+View current backups:
+
+.. code-block:: bash
+
+   pgbackrest info --stanza=example
+
+Restore from backup:
+
+.. code-block:: bash
+
+   pgbackrest restore --stanza=example --delta
+
+Restore specific backup by timestamp:
+
+.. code-block:: bash
+
+   pgbackrest restore --stanza=example --set=20210315-145357F_20210315-145459I --delta
+
```

Comment:
@jpmckinney I've rewritten this paragraph in the docs. Aiming to address _this comment https://github.com/open-contracting/deploy/pull/254#discussion_r605267107_.

How does the below read?

Reply:
Yes, it's clearer!